stanford/playbooks/certs.yml

```diff
@@ -10,6 +10,7 @@
   become: True
   vars:
     CLUSTER_NAME: certs
+    INSTALL_INSTANCES: true
```

Comment:
See comments below about removing these declarations,
after considering this to the be default case.